docs/guides/migrating-from-camunda-7/conceptual-differences.md

```diff
@@ -46,7 +46,21 @@ To migrate existing Connectors, consider the following options:
 - Use the [REST protocol Connector](components/connectors/protocol/rest.md) to leverage an out-of-the-box Connector.
 - Create a small bridging layer via custom [job workers](/components/concepts/job-workers.md).
 
-### Process solutions using Spring Boot
+### Multi-tenancy
+
+:::note
+[Multi-tenancy](/self-managed/concepts/multi-tenancy.md) is currently in development for Camunda 8.
+:::
+
+There are several differences between how [multi-tenancy](/self-managed/concepts/multi-tenancy.md) works in Camunda 7 and Camunda 8:
+
+1. The one engine per tenant approach from Camunda 7 isn't possible with Camunda 8. Camunda 8 only provides multi-tenancy through a tenant identifier.
+2. In Camunda 7, users can deploy shared resources (processes, decisions, and forms) to all tenants. In Camunda 8, there are no shared resources. This will be added in the future.
+3. In Camunda 7, data is mapped to a `null` tenant identifier, meaning by default resources are shared. In Camunda 8, data is mapped to the `<default>` tenant identifier when multi-tenancy is disabled.
+4. Tenant checks in Camunda 7 can be disabled to perform admin/maintenance operations. This can't be done in Camunda 8, but an admin user can be authorized to all tenants, which would result in the same thing.
+5. Users in Camunda 7 that are authorized for multiple tenants may perform a lot more operations without providing a `tenantId`. The Camunda 7 engine will try to infer the correct `tenantId` as much as possible. Tenant inference in the Zeebe broker doesn't happen in Camunda 8, and ask users to provide the `tenantId` explicitly.
```

Comment:
:x: This point may be confusing to C7 users, as we never explicitly define "tenant inference" in the C7 docs. We only mention that if a user tries to trigger a command on a resource mapped to multiple tenants, an exception is thrown, and the `tenantId` must be implicitly provided (see subsections [here](https://docs.camunda.org/manual/develop/user-guide/process-engine/multi-tenancy/#run-commands-for-a-tenant)).

This implies tenant inference exists, but we only document the case when tenant inference isn't possible.

Reply:
I adjusted in my latest commit, WDYT?